src/content/it/2020/resource-hints.md

```diff
@@ -0,0 +1,461 @@
+discuss: 2057
+results: https://docs.google.com/spreadsheets/d/1lXjd8ogB7kYfG09eUdGYXUlrMjs4mq1Z7nNldQnvkVA/
+queries: 21_Resource_Hints
+featured_quote: During the past year resource hints increased in adoption, and they have become essential APIs for developers to have more granular control over many aspects of resource prioritizations and ultimately, user experience.
+featured_stat_1: 33%
+featured_stat_label_1: Sites using <code>dns-prefetch</code>
+featured_stat_2: 9%
+featured_stat_label_2: Sites using <code>preload</code>
+featured_stat_3: 4.02%
+featured_stat_label_3: Sites using native lazy loading
+---
+
+## Introduzione
+
+<!-- Over the past decade [resource hints](https://www.w3.org/TR/resource-hints/) have become essential primitives that allow developers to improve page performance and therefore the user experience. -->
+
+Negli ultimi dieci anni [resource hints](https://www.w3.org/TR/resource-hints/) sono diventati primitivi essenziali che consentono agli sviluppatori di migliorare le prestazioni della pagina e quindi la user experience.
+
+<!-- Preloading resources and having browsers apply some intelligent prioritization is something that was actually started way back in 2009 by IE8 with something called the [preloader](https://speedcurve.com/blog/load-scripts-async/). In addition to the HTML parser, IE8 had a lightweight look-ahead preloader that scanned for tags that could initiate network requests (`<script>`, `<link>`, and `<img>`). -->
+
+Il precaricamento delle risorse e il fatto che i browser applichino una priorità intelligente è qualcosa che è stato effettivamente avviato nel lontano 2009 da IE8 con qualcosa chiamato [preloader](https://speedcurve.com/blog/load-scripts-async/). Oltre al parser HTML, IE8 aveva un preloader leggero che analizzava i tag in grado di avviare richieste di rete (`<script>`, `<link>` e `<img>`).
+
+<!-- Over the following years, browser vendors did more and more of the heavy lifting, each adding their own special sauce for how to prioritize resources. But it's important to understand that the browser alone has some limitations. As developers however, we can overcome these limits by making good use of resource hints and help decide how to prioritize resources, determining which should be fetched or preprocessed to further boost page performance. -->
+
+Negli anni successivi, i fornitori di browser hanno svolto sempre più il lavoro pesante, ognuno aggiungendo la propria salsa speciale su come dare la priorità alle risorse. Ma è importante capire che il browser da solo ha alcune limitazioni. In qualità di sviluppatori, tuttavia, possiamo superare questi limiti facendo un buon uso delle resource hints e aiutandoci a decidere come assegnare la priorità alle risorse, determinando quali dovrebbero essere fetched o preprocessed per aumentare ulteriormente le prestazioni della pagina.
```

Comment:
Dire `più il lavoro pesante` suona strano, forse possiamo dire `i fornitori di browser hanno svolto sempre più il grosso del lavoro,`. Che dici?

Reply:
@chefleo attento nel riposizionare gli aggettivi correttamente, invece di `che il browser da solo` dire `che da solo il browser`. Praticamente rileggi ad alta voce tutto il documento e fai in modo che l'Italiano suoni naturale e non tradotto da Google Translate 😉